translation/file_loader_test.go

```diff
@@ -30,10 +33,15 @@ func TestFileLoaderTestSuite(t *testing.T) {
 }
 
 func (f *FileLoaderTestSuite) SetupTest() {
+	f.app = &foundation.Application{}
 }
 
 func (f *FileLoaderTestSuite) TestLoad() {
-	paths := []string{"./lang"}
+	f.app.On("ExecutablePath").Return("./", nil)
+	executable, err := f.app.ExecutablePath()
+	assert.NoError(f.T(), err)
```

Comment:
Actually, the mock is unnecessary, because we test `NewFileLoader` directly, don't use `App`. And I mean to test a real executable path, instead of `./`, you can use `os` to get the executable path directly, then test `NewFileLoader(paths)` and `loader.Load` are normal.

Reply:
> Actually, the mock is unnecessary, because we test `NewFileLoader` directly, don't use `App`. And I mean to test a real executable path, instead of `./`, you can use `os` to get the executable path directly, then test `NewFileLoader(paths)` and `loader.Load` are normal.

added, but seems a bit strange.